go/src/koding/kites/kloud/provider/aws/default.json

```diff
@@ -0,0 +1,20 @@
+{
+  "provider": {
+    "aws": {
+      "access_key": "${var.aws_access_key}",
+      "secret_key": "${var.aws_secret_key}"
+    }
+  },
+  "resource": {
+    "aws_instance": {
+      "aws-instance": {
+        "ami": "",
+        "instance_type": "${var.userInput_instance_type}",
```

Comment:
There will be a backend for it in kloud, so by default kloud is going to fill it with defaults.

The backend will also have an switch for pass raw templates (with userInput) to kd, so kd can have the interactive `kd init`.